<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1561</width>
      <height>841</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="widget_0" native="true">
       <widget class="QPushButton" name="devOpenButton_0">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>10</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Open Device</string>
        </property>
       </widget>
       <widget class="QPushButton" name="camStartButton_0">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>50</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_0">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>3</y>
          <width>641</width>
          <height>411</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_0">
         <item>
          <widget class="QLabel" name="colorLabel_0">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="depthLabel_0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maskLabel_0">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QRadioButton" name="standaloneRadio_0">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>110</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>STANDALONE</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="masterRadio_0">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>140</y>
          <width>112</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>MASTER</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="subRadio_0">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>170</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>SUBORDINATE</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>3</y>
          <width>641</width>
          <height>411</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="colorLabel_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="depthLabel_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maskLabel_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="devOpenButton_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>10</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Open Device</string>
        </property>
       </widget>
       <widget class="QPushButton" name="camStartButton_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>50</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="standaloneRadio_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>110</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>STANDALONE</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="masterRadio_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>140</y>
          <width>112</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>MASTER</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="subRadio_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>170</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>SUBORDINATE</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QWidget" name="widget_3" native="true">
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>3</y>
          <width>641</width>
          <height>411</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="colorLabel_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="depthLabel_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maskLabel_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="devOpenButton_3">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>10</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Open Device</string>
        </property>
       </widget>
       <widget class="QPushButton" name="camStartButton_3">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>60</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="standaloneRadio_3">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>110</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>STANDALONE</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="masterRadio_3">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>140</y>
          <width>112</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>MASTER</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="subRadio_3">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>170</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>SUBORDINATE</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QWidget" name="widget_1" native="true">
       <widget class="QWidget" name="layoutWidget_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>3</y>
          <width>641</width>
          <height>411</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <item>
          <widget class="QLabel" name="colorLabel_1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="depthLabel_1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maskLabel_1">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="devOpenButton_1">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>10</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Open Device</string>
        </property>
       </widget>
       <widget class="QPushButton" name="camStartButton_1">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>50</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="standaloneRadio_1">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>100</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>STANDALONE</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="subRadio_1">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>160</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>SUBORDINATE</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="masterRadio_1">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>130</y>
          <width>112</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>MASTER</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionCamera_Settings">
   <property name="text">
    <string>Camera Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
